packages/protocol/contracts/tokenvault/BaseNFTVault.sol

```diff
@@ -33,14 +33,11 @@ abstract contract BaseNFTVault is EssentialContract {
         uint256[] tokenIds;
         uint256[] amounts;
         uint256 gasLimit;
-        uint256 processingFee;
-        address refundAddress;
+        uint256 fee;
```

Comment:
This contract  and ERC20Vault should inherit the IRecallableMessageSender interface